pkg/shell/superuser.jsx

```diff
@@ -180,6 +191,33 @@ const UnlockDialog = ({ proxy, host }) => {
             <Button variant="secondary" className='btn-cancel' onClick={cancel}>
                 {_("Close")}
             </Button>);
+    } else if (methods) {
+        const apply = () => {
+            start(method);
+        };
```

Comment:
Useless `{};` wrapper. `const apply = () => start(method);`

Reply:
Fixed by removing `apply` completely, it was only used once.